client/src/components/TabBody/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Textarea from '../Textarea'
+import './style.css'
+
+export default function TabBody(props) {
+
+  const { href, question, tips, expectations, onChange, wordsLimit } = props;
+
+  return (
+    <container className='tabBody'>
+      <div className='tabBody-guidelines'><h2>Refer to <strong><a className='tabBody-guidelines-href' href={href}>application guidelines</a></strong></h2></div>
```

Comment:
we can change the font-weight in the css of .tabBody-guidelines-href rather tan adding the strong tag. It'll give us more options in regards styling.

Reply:
Done